docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installation
+
+- [Installation](#installation)
+  - [Provision of External IP for LoadBalancer service type in Kubernetes](#provision-of-external-ip-for-loadbalancer-service-type-in-kubernetes)
+    - [Minikube](#minikube)
```

Comment:
Maybe add k3s/k3d and simply say it's supported out of the box?

Reply:
Done.